src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -172,31 +172,32 @@ static ucs_status_t uct_ib_mlx5_devx_reg_ksm_data_addr(
 
 static ucs_status_t
 uct_ib_mlx5_devx_reg_ksm_data_contig(uct_ib_mlx5_md_t *md,
-                                     uct_ib_mlx5_devx_mr_t *mr, off_t offset,
-                                     void *address, int atomic,
-                                     const char *reason,
+                                     uct_ib_mlx5_devx_mr_t *mr, void *address,
+                                     void *iova, int atomic, const char *reason,
```

Comment:
iova param better be uint64_t, to align with verbs API

Reply:
uct_ib_mlx5_devx_reg_ksm_data_addr accept it as `uintptr_t`, to change it there as well?